### ISSUE TITLE
Show focus ring set to false, ring still here

### ISSUE THREAD

**Issue by BeBel42** [mkocansey/bladewind#208]
Hi,

This issue is in fact two issues:

Here is my actual button:
```html
<x-bladewind.button show_focus_ring='false' button_text_css='font-black' border_width='2' uppercasing='false' outline='true' size='small' tag='a' href="{{ route('manage.images') }}">Manage Images</x-bladewind.button>
```

I decreased the border_width, and disabled the focus ring.
The focus ring still shows up, even though I disabled it:

![2024-03-14T20:58:18:002330067](https://github.com/mkocansey/bladewind/assets/64377698/a353b7bd-5c49-428a-a8dc-3947e3d6c06e)

And for the second issue, the focus ring is a bit too thick, but the main issue is that it is white, and not a shade of the button's color.

**Comment by BeBel42** [comment]
I have found where the issue is from.

It is from the bw-button class. I cannot edit this because it is in a .min.css file.
The white I see is a box-shadow. It would be great to make it dark when dark theme is enabled in tailwind, with `dark:`.
I am wondering, why would that button class be defined in

**Comment by mkocansey** [comment]
Do let me know if you’re still experiencing this in v2.5.2?

**Comment by mkocansey** [comment]
@BeBel42 will push a fix for this soon

**Comment by BeBel42** [comment]
> Do let me know if you’re still experiencing this in v2.5.2?

Yes it still does that

**Comment by mkocansey** [comment]
@BeBel42 
Okay.. fixed but working on a few other fixes and I'll push an update. 

To answer your earlier question... 
I maintain the classes for the button and a few other components in an external css file because it's easier to manage. Those specific components have a lot of conditions that make maintaining the classes inline quite tedious.

**Comment by mkocansey** [comment]
@BeBel42 this is fixed in 2.5.3. 

`composer update`
